### PR TITLE
pydocstyle 6.1.1 (new formula)

### PR DESCRIPTION
**Pull request by singingwolfboy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pydocstyle.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class Pydocstyle < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python docstring style checker"
+  homepage "https://www.pydocstyle.org/"
+  url "https://github.com/PyCQA/pydocstyle/archive/refs/tags/6.1.1.tar.gz"
```
Comment (nandahkrishna) [inline Formula/pydocstyle.rb:6]: Can we use the source tarball available from [PyPI](https://pypi.org/project/pydocstyle/#files)? That would make tracking resources easier (and automatic with `bump-formula-pr`/`update-python-resources`).

Comment (singingwolfboy) [reply Formula/pydocstyle.rb]: Done! Ready for another review

**Review by nandahkrishna — COMMENTED** [review]
We should modify `pypi_formula_mappings.json` like this so that the `toml` resource, which is an extra (according to `setup.py`), isn't removed:
```diff
diff --git a/pypi_formula_mappings.json b/pypi_formula_mappings.json
index 0502484be9..744dd309a3 100644
--- a/pypi_formula_mappings.json
+++ b/pypi_formula_mappings.json
@@ -140,6 +140,7 @@
   "protobuf": {
     "exclude_packages": ["six"]
   },
+  "pydocstyle": "pydocstyle[toml]",
   "pyqt-builder": {
     "exclude_packages": ["packaging", "pyparsing", "sip", "toml"]
   },
```

**Comment by singingwolfboy** [comment]
Good call. Fixed!

**Comment by carlocab** [comment]
Thanks, @singingwolfboy. Can you squash your commits together into the first one? BrewTestBot can't merge PRs that modify non-formula files.

**Comment by singingwolfboy** [comment]
Squashed

**Review by nandahkrishna — APPROVED** [review]
Thanks!

**Pull request closed without merging**